Sources/Stores/NetworkPersistableStore.swift

```diff
@@ -242,8 +242,7 @@ where Network.Remote == Data, Network.Request == URLRequest, Network.Response ==
             switch result {
             case .success(let data):
                 success(data)
-            case .failure(.url(let error as NSError))
-            where error.domain == NSURLErrorDomain && error.code == NSURLErrorCancelled:
+            case .failure(.url(let error as NSError)) where error.isURLErrorCancelled:
```

Comment:
Just out of curiosity. Couldn't you switch on the error type instead? I.e.
``` swift
case .failure(.url(URLError.cancelled))
```

Reply:
Great point! Even though we can (and probably should) use `URLError`, I think that we cannot switch on `URLError.cancelled`, as that's only the code of the error 😓. Still, I think we can go with:

`case .failure(.url(let error as URLError)) where error.code == .cancelled:`

😉